App/src/Entity/Bilan.php

```diff
@@ -2,12 +2,16 @@
 
 namespace App\Entity;
 
+
 class Bilan
 {
     protected string $name;
     protected int $cost = 0;
     protected int $participation = 0;
     protected int $balance = 0;
+    /**
```

Comment:
Tu peux séparer tes propriétés qui ont du PHPDOC avec un saut de ligne supplémentaire 👍 
```suggestion

    /**
```

Reply:
done